docs/sources/features/explore/index.md

```diff
@@ -274,3 +274,10 @@ Simply clicking the button will return you to the origin dashboard, or, if you'd
 the arrow next to the button to reveal a "Return to panel with changes" menu item.
 
 {{< docs-imagebox img="/img/docs/v64/explore_return_dropdown.png" class="docs-image--no-shadow" caption="Screenshot of the expanded explore return dropdown" >}}
+
+## Query Inspector
+
+To help with debugging queries, Explore allows users to investigate query requests and responses, as well as query statistics,
+via the Query Inspector. You can drill down into specific portions of the query, expand or collapse all of it, or copy the data to the clipboard to use in other applications.
```

Comment:
- Unclear what "drill down" means in this context. Please clarify.
- Best practice is to avoid sentences longer than 25 words. Please break this up. 
- How do I do any of the things you are talking about? I don't see instructions. Is any of it covered by the [Panel inspector instructions](https://grafana.com/docs/grafana/latest/panels/inspect-panel/)? If so, you can link to the section.

Reply:
"Drill down" is used in the documentation for the panel inspector: https://grafana.com/docs/grafana/latest/panels/inspect-panel/#view-raw-request-and-response-to-data-source